### DISCUSSION TITLE
Change rebase from "master" to "main"

### DISCUSSION THREAD

**Discussion by guevarez30** [NeogitOrg/neogit#434] [General]
Current git repo use main as the default branch nomenclature. Is there a way to change the default branch from "master" to "main"

```
Actions
p Rebase onto master    
e Rebase onto elsewhere 
i Interactive  
```

Error:
```
git --no-pager -c color.ui=always --no-optional-locks rebase master:
    fatal: invalid upstream 'master'
```

**Comment by CKolkey** [comment]
Something like this? https://github.com/TimUntersberger/neogit/pull/464
